papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,498 @@
+-------------------------------------------------------------------
+
+.. class:: abstract
+
+The signac data management framework (https://signac.io) helps researchers execute reproducible computational studies, scaling from laptops to supercomputers and emphasizing portability and fast prototyping.
+With signac, users can track, search, and archive data and metadata for file-based workflows and automate workflow submission on high performance computing (HPC) clusters.
+We will discuss recent improvements to the software's feature set, scalability, scientific applications, usability, and community.
+Newly implemented synced data structures, features for generalized workflow execution, and performance optimizations will be covered, as well as recent research using the framework and the project's efforts on improving documentation, contributor onboarding, and governance.
+
+.. class:: keywords
+
+   data management, data science, database, simulation, collaboration, workflow, HPC, reproducibility
+
+
+Introduction
+------------
+
+.. figure:: signac_overview.pdf
+    :align: center
+    :scale: 40 %
+    :figclass: w
+
+    Overview of the signac framework.
+    Users first create a project, which initializes a workspace directory on disk.
+    Users define state points which are dictionaries that uniquely identify a job.
+    The workspace holds a directory for each job, containing JSON files that store the state point and job document.
+    The job directory name is a hash of the state point's contents.
+    Here, the ``init.py`` file initializes an empty project and adds one job with state point ``{"a": 1}``.
+    Next, users define a workflow using a subclass of signac-flow's ``FlowProject``.
+    The workflow shown has three operations (simulate, analyze, visualize) that, when executed, produce two new files ``results.txt`` and ``plot.png`` in the job directory.
+
+The full source code of all examples in this paper can be found online [#]_.
+
+.. [#] https://github.com/glotzerlab/signac-examples
+
+Research projects often address problems where questions change rapidly, data models are always in flux, and compute infrastructure varies widely from project to project.
+The signac data management framework is a tool designed by researchers, for researchers, to make prototyping quick and reproducibility easy.
+It forgoes serializing complex data files into a database in favor of working with these files directly, providing fast indexing utilities for a set of directories.
+Using signac, a data space on the file system is easily initialized, searched, and modified using either a Python or command-line interface.
+The companion package signac-flow interacts with the data space to generate and analyze data through reproducible workflows that easily scale from laptops to supercomputers.
+signac-flow can run arbitrary commands as part of a workflow, making it as flexible as a script in any language of choice.
```

Comment:
This sentence needs to be re-worked. We try to avoid the problem of questionable capitalization by not starting sentences with "signac" or "signac-flow." There is (or was) at least one other instance of this where a sentence starts with "Signac" (capitalized). I have a suggestion below but there might be a better way.
```suggestion
Arbitrary shell commands can be run by signac-flow as part of a workflow, making it as flexible as a script in any language of choice.
```